src/library/Cardano/CoinSelection/Migration.hs

```diff
@@ -8,11 +8,10 @@
 -- Copyright: © 2018-2020 IOHK
 -- License: Apache-2.0
 --
--- This module contains an algorithm to select coins for migration from legacy
--- wallets to newer wallets.
+-- This module contains an algorithm to select coins for emptying a wallet.
```

Comment:
I think it would be better to use the word "migration" here, as "emptying" has the possible connotation that funds would be thrown away (when in fact, we want them to be _migrated_ _somewhere else_).

How about:
```suggestion
-- This module contains an algorithm for migrating all funds away from a wallet.
```